test/langtools/jdk/internal/shellsupport/doc/JavadocFormatterTest.java

```diff
@@ -137,7 +137,7 @@ public void testReflow() {
                    "1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234\n" +
                    "1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234\n" +
                    "1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234 1234\n" +
-                   "1234 1234 1234 1234 1234 1234 1234 1234 1234\n";
+                   "1234 1234 1234 1234 1234 1234 1234 1234 1234 \n";
```

Comment:
What's that about? I mean is it related to this work on `{@return}`?

Reply:
It was needed to fix a test failure

Reply:
FWIW, I believe the reason is that JavadocFormatter.scan (around line 580 in the new code) is checking for inline tags using the instanceof operator, and adds a space for inline tags. We could add a test to prevent that (i.e. not add the space , but not sure if there is a real problem with generating the space.